en/integrations/plugin-packs/procedures/network-fortinet-fortiweb-snmp.md

```diff
@@ -0,0 +1,168 @@
+## Host configuration
+
+* Add a new Host and fill the *IP Address/FQDN*, *SNMP Version* and *SNMP Community* fields according to the device's configuration
+* Apply the *Net-Fortinet-Fortiauthenticator-SNMP-custom* Host Template
+
+> When using SNMP v3, use the SNMPEXTRAOPTIONS Macro to add specific authentication parameters
+
+| Mandatory | Name             | Description                                    |
+| :-------- | :--------------- | :--------------------------------------------- |
+|           | SNMPEXTRAOPTIONS | Configure your own SNMPv3 credentials combo    |
+
+## How to test the Plugin and what are the main options for?
+
+Once the plugin installed, log into your Centreon Poller CLI using the *centreon-engine* user account
+and test the Plugin by running the following command:
+
+```bash
+/usr/lib/centreon/plugins/centreon_fortinet_fortiweb_snmp.pl \
+    --plugin=network::fortinet::fortiweb::snmp::plugin \
+    --mode=system \
+    --hostname=10.30.2.114 \
+    --snmp-version='2c' \
+    --snmp-community='fortinet_ro' \
+    --warning-cpu-load='90' \
+    --critical-cpu-load='95' \
+    --verbose
+```
+
+Expected command output is shown below:
+
+```bash
+OK: system usage is ok | 'cpu.utilization.percentage'=12.00%;0:90;0:95;0;100 'memory.usage.percentage'=79.00%;;;0;100 'disk.log.space.usage.percentage'=74.00%;;;0;100
+checking system
+    high-availability mode: standalone
+    cpu load: 12.00 %
+    memory used: 79.00 %
+    disk log space used: 74.00 %
+```
+
+The command above monitors Fortinet FortiWeb (```--plugin=network::fortinet::fortiweb::snmp::plugin --mode=system```) identified
+by the IP address *10.30.2.114* (```--hostname=10.30.2.114```). As the Plugin is using the SNMP protocol to request the device, the related
+*community* and *version* are specified (```--snmp-version='2c' --snmp-community='fortinet_ro'```).
+
+This command would trigger a WARNING alarm if cpu utilization over 90% 
+(```--warning-cpu-utilization='90'```) and a CRITICAL alarm over 95% (```--critical-cpu-utilization='95'```).
+
+All the options as well as all the available thresholds can be displayed by adding the  ```--help```
+parameter to the command:
+
+```bash
+/usr/lib/centreon/plugins/centreon_fortinet_fortiweb_snmp.pl \
+    --plugin=network::fortinet::fortiweb::snmp::plugin \
+    --mode=system \
+    --help
+```
+
+## Troubleshooting
+
+### UNKNOWN: SNMP GET Request : Timeout
+
+If you get this message, you're probably facing one of theses issues:
+* The SNMP agent of the device isn't started or is misconfigured
+* An external device is blocking the request (firewall, ...)
+
+#### UNKNOWN: SNMP GET Request : Cant get a single value.
+
+This error message often refers to the following issues: 
+  - The agent doesn't support the MIB used by the plugin
+  - The targeted SNMP OID cannot be fetched because of insufficient privileges on the device. 
+    SNMP Agent must be capable of accessing to the enterprise branch: .1.3.6.1.4.1.12356.107
```

Comment:
Don't know if we should keep this one. I agree that it adds relevant information on top of the standard procedure.